### PR TITLE
feature(improve-dependencies): Improve Dependencies and Resolution

### PR DESCRIPTION
**Pull request by ryanbas21**
Purge dependencies and remove as many duplicates and things we don't
need. Clean package-json :smiles

# JIRA Ticket

Please link jira ticket here

# Description

# Type of Change

Please Delete options that are not relevant

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Definition of Done

Check all that apply

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Code peer-reviewed.
- [ ] Ensure backward compatibility (special attention).
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] CI build passing on the feature branch.
- [ ] Functional spec is written/updated
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] UI is completed or ticket is created.
- [ ] Demo to PO and team.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

## Documentation

- [ ] Acceptance criteria met
- [ ] Spell-check run
- [ ] Peer reviewed
- [ ] Proofread

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,62 +36,45 @@
...
+    "@nrwl/nx-cloud": "13.1.6",
+    "@nrwl/react": "13.8.8",
     "@nrwl/schematics": "^8.12.11",
-    "@nrwl/tao": "^13.3.9",
-    "@nrwl/web": "^13.3.9",
-    "@nrwl/workspace": "^13.3.9",
-    "@playwright/test": "^1.17.1",
+    "@nrwl/tao": "13.8.8",
+    "@nrwl/web": "13.8.8",
+    "@nrwl/workspace": "13.8.8",
```
Comment (cerebrl) [inline package.json:56]: A lot of this looks like the same changes from this PR: https://github.com/ForgeRock/forgerock-javascript-sdk/pull/188/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L111-L113. Is there a shared commit or something?

**Review by cerebrl — COMMENTED** [review]
I'll run this locally tomorrow, but it does feel like there are some shared commits between other PRs.

File: package.json (hunk 1 above)
Comment (ryanbas21) [reply package.json:56]: Correct. I didn't want to deal with the rebase. I wrote in that PR that this is required. This is the PR that aligns with sample app dependency clean up just in our develop

**Comment by daveadams56** [comment]
Just thinking about the other PRs we have open @ryanbas21 - should we add the same npm scripts to run the sample apps that we have in the start and complete branches? I noticed that we need to run them using Nx, in this branch

**Comment by ryanbas21** [comment]
> Just thinking about the other PRs we have open @ryanbas21 - should we add the same npm scripts to run the sample apps that we have in the start and complete branches? I noticed that we need to run them using Nx, in this branch

I'm actually adding them in the open PR that requires docs changes. That way everything will sync with documentation and new commands.

**Review by daveadams56 — COMMENTED** [review]
Build, test, lint and sample apps ran ok - and these look sensible to me. I'll be happy to approve once we've discussed the two points I've commented on

File: samples/reactjs-todo/webpack.config.js [hunk 2]
```diff
@@ -30,8 +30,6 @@ module.exports = (config) => {
 
     devServer: {
       ...config.devServer,
-      host: '0.0.0.0',
```
[the remark this answers is not in the document]
Comment (ryanbas21) [reply samples/reactjs-todo/webpack.config.js:33]: It was erronous from when I was working within the docker container. Docker containers need this for webpack devServer to run correctly.

As i've been post poned on the preview env work, we don't need that right now. It was from when I merged the docker compose work.

[the remark this answers is not in the document]
Comment (daveadams56) [reply samples/reactjs-todo/webpack.config.js:33]: Ok sounds good, thanks

**Pull request merged**